client/src/components/DataFiles/DataFilesTable/DataFilesTable.js

```diff
@@ -18,11 +18,16 @@ import './DataFilesTable.module.scss';
 // What to render if there are no files to display
 const DataFilesTablePlaceholder = ({ section, data }) => {
   const dispatch = useDispatch();
+  const currentListing = useSelector(
```

Comment:
suggestion/nitpick that can be ignored:  maybe `currentListingApi` to be more clear?